src/recordings.js

```diff
@@ -47,71 +55,41 @@ export function retrieveListOfSpeakers() {
-
-    // generate a new link and append it to the page if there isn't already one
-    if (container.childElementCount < 1) {
-      // clone our template so we can insert it into the DOM
-      let createdLink = SimpleTemplate.fromId('template:speaker-bio-link')
-
-      // set the speaker-link text with the name of the speaker
-      createdLink.slot['speaker-name'] = speakerName
-
-      // set the link URL
-      createdLink.element.href = insertedURL
-
-      // and place the node into the DOM
-      container.appendChild(createdLink.element)
-    } else {
-      // remove the node that was created:
-      container.removeChild(container.childNodes[1]) // may need to extract the inner parameter based on Eddie's feedback...
-
-      // create a new node for the new speaker name
-      let newSpeakerNode = SimpleTemplate.fromId('template:speaker-bio-link')
-      // ...and place the newly clicked speaker's name into it
-      newSpeakerNode.slot['speaker-name'] = speakerName
-
-      // get the URL again and reinsert into the newly created node
-      newSpeakerNode.element.href = insertedURL
-
-      // place said node into the DOM
-      container.appendChild(newSpeakerNode.element)
-    }
+        for (let recordingData of recordings) {          
+          // TODOkobe: if all is working/Eddie has approved, perhaps refactor into a function?
+          // create a option element
+          let listOption = document.createElement('option')
+          
+          // create a new textnode that is the speaker's name
+          let nodeSpeakerName = document.createTextNode(recordingData.speaker_name + ', ' + recordingData.dialect)
+    
+          // place the newly created node into the option element...
+          listOption.appendChild(nodeSpeakerName)
+    
+          // ...and insert the option element into the dropdown list 😌
+          recordingsDropdown.appendChild(listOption)
+          
+          // TODOkobe: if there was to be a check for "if there's speakers with the same name, add something to the way they're rendered", it would end up right here as a loop within this loop,,,
+        }
+      
+        // audio playback for the specific speaker
+        recordingsPlayback.addEventListener('click', () => {
+          let speakerPosition = recordingsDropdown.selectedIndex
+          let audioURLPosition = recordings[speakerPosition].recording_url
+
+          // play the audio associated with that specific index
+          let audio = new Audio(audioURLPosition) // poorly named...position isn't it 💀
+          audio.preload = 'none'
+          audio.play()
+        })
+
+        // link for the specific speaker
+        recordingsLink.addEventListener('click', () => {
+          let speakerPosition = recordingsDropdown.selectedIndex
+          let speakerBioLink = recordings[speakerPosition].speaker_bio_url
+          
+          // open the speaker's link in a new tab!
+          window.open(speakerBioLink, '_blank')
+        })
```

Comment:
Now that I think of it, perhaps this should just be a `<a target="_blank">` in the HTML, and require zero JavaScript code. We can use CSS to make the link look like the button, but act like a link.

Reply:
So I ended up going with a text link over the SVG icon: will that work? I figured it knocked out one of the pieces of design feedback (not immediately understanding what the external link button did).

I can always revert back to the SVG if you think that's better, though!

Reply:
Yes, text works for this!